src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random;
+
+    ColorSupplier() {
+        this.random = new Random();
+    }
+
+    public Color getRandomColor() {
+        Color [] colors = Color.values();
```

Comment:
Indentation issue :)
```suggestion
    Color[] colors = Color.values();
```